src/main/kotlin/no/nav/syfo/personstatus/api/v2/model/PersonInfoDTO.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.syfo.personstatus.api.v2.model
+
+import no.nav.syfo.personstatus.domain.PersonOversiktStatus
+
+data class PersonInfoDTO(
+    val veilederIdent: String?,
+    val fnr: String,
+    val navn: String?,
+    val enhet: String?,
```

Comment:
Foreslår `personident` i stedet for `fnr`. Trenger vi å sende navn på personen? Tror vi må hente veiledernavn i frontenden fra `syfoveileder`, men det har vi en hook på allerede 👍🏼

Reply:
Tenkte `navn` var en del av den grunnleggende informasjonen om en bruker som jeg tenkte denne DTO'en representerte. Kan droppe til vi ser at vi har behov for det 😊